vector/v.lidar.growing/v.lidar.growing.html

```diff
@@ -89,11 +80,13 @@ <h2>REFERENCES</h2>
 <p>
 Performances of the filter can be seen in the
 <a href="http://www.itc.nl/isprswgIII-3/filtertest/MainDoc.htm">ISPRS WG III/3 Comparison of Filters</a> 
-report by Sithole, G. and Vosselman, G., 2003. 
-<br>
+report by Sithole, G. and Vosselman, G., 2003.
 
-<!--
+<h2>AUTHORS</h2>
+Original version of program in GRASS 5.4:
+<br>
+Maria Antonia Brovelli, Massimiliano Cannata, Ulisse Longoni and Mirko Reguzzoni
 <p>
-<i>Last changed: $Date$</i>
--->
-
+Update for GRASS 6.X:
+<br>
+Roberto Antolin and Gonzalo Moreno
```

Comment:
`<h2>REFERENCES</h2>` section should be before `<h2>SEE ALSO</h2>` section.

`<h2>EXAMPLES</h2>` should be `<h2>EXAMPLE</h2>`.